client/src/api/pastes.js

```diff
@@ -26,10 +26,26 @@ export function getRandomPaste() {
 
 export function postPaste(paste) {
   return fetch(`/api/pastes`, {
-    method: "POST",
+    method: 'POST',
     headers: {
-      "Content-Type": "application/json"
+      'Content-Type': 'application/json'
     },
     body: JSON.stringify(paste)
   }).then(response => response.json());
 }
+
+// Could be in a seperate file =)
+const SLACK_URL =
+  'https://hooks.slack.com/services/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';
```

Comment:
We have to change this to 'https://hooks.slack.com/services/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 '